pkg/chains/signing.go

```diff
@@ -224,3 +244,41 @@ func (ts *TaskRunSigner) SignTaskRun(ctx context.Context, tr *v1beta1.TaskRun) e
 	// Now mark the TaskRun as signed
 	return MarkSigned(tr, ts.Pipelineclientset)
 }
+
+type rekor struct {
```

Comment:
We could move the rekor stuff into `pkg/chains/rekor` so it's easy to find

Reply:
or, actually, if that's annoying for testing maybe just move it into a `rekor.go` file!

Reply:
Done!